Makefile

```diff
@@ -160,9 +160,8 @@ clean-doc:
 	$(RM) -r doc/xhtml 
 	$(RM) doc/*.xml doc/*.1 doc/newsboat-cfgcmds.asciidoc \
 		doc/podboat-cfgcmds.asciidoc doc/newsboat-keycmds.asciidoc \
-		doc/configcommands-linked.dsv doc/podboat-cmds-linked.dsv \
-		doc/keycmds-linked.dsv doc/gen-example-config \
-		doc/example-config doc/generate doc/generate2
+		doc/example-config doc/generate doc/generate2 \
+		doc/gen-example-config
```

Comment:
I think the new `*-linked.asciidoc` files should be added here, so one can remove them with a single `make clean`. (Sorry I didn't think of this when I asked to add them to .gitignore!)

Reply:
Do the lines stay in the .gitignore file?

Reply:
Yes. They work in concert: `make distclean` (which includes `clean-doc`) ensures a "clean slate" for any work that follows (like a build with `make all`), while `.gitignore` ensures that these temporary files stay out of `git status` and aren't accidentally added to the repository.